### PR TITLE
Add bash completion

### PR DESCRIPTION
**Pull request by lachmanfrantisek**
Signed-off-by: lachmanfrantisek <flachman@redhat.com>

I've seen the bash completion in the click docs, so why not have that.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -21,7 +21,8 @@
     ''',
     data_files=[("share/colin/config/", ["config/default.json",
                                          "config/fedora.json",
-                                         "config/redhat.json"])],
+                                         "config/redhat.json"]),
+                ("/usr/share/bash-completion/completions/colin", ["bash-complete"])],
```
[the remark this answers is not in the document]
Comment (lachmanfrantisek) [reply setup.py]: On my Fedora 27 also in `/usr/share/bash-completion/completions/colin`..:-)

[the remark this answers is not in the document]
Comment (jpopelka) [reply setup.py]: +1 to `/usr/share/bash-completion/completions/colin`

```
# pkg-config --variable=completionsdir bash-completion
/usr/share/bash-completion/completions
```

[the remark this answers is not in the document]
Comment (lachmanfrantisek) [reply setup.py]: `~/.local/share/bash-completion/completions/` works as well

File: setup.py [hunk 2]
```diff
@@ -21,7 +21,8 @@
     ''',
     data_files=[("share/colin/config/", ["config/default.json",
                                          "config/fedora.json",
-                                         "config/redhat.json"])],
+                                         "config/redhat.json"]),
+                ("share/bash-completion/completions/", ["colin-bash-complete"])],
```
Comment (jpopelka) [inline setup.py]: I liked just `colin` more. None of the files in `/usr/share/bash-completion/completions` have `bash` or `complete` in name.

Comment (lachmanfrantisek) [reply setup.py]: Yes, the previous version creates the `colin` directory in `..../completions` and place there the file.

Is there some way, how to rename the file in setup.py, or I have to name it colin in the repository?

**Comment by lachmanfrantisek** [comment]
Thanks!

**Pull request merged**